src/AppBanner/index.tsx

```diff
@@ -33,21 +48,19 @@ export type Props = {
 };
 
 export const AppBanner = (props: Props): ReactElement | null => {
+    if (!canRender(props.brazeMessageProps)) {
+        return null;
+    }
+
     const {
         brazeMessageProps: { ophanComponentId = COMPONENT_NAME, header, body, cta, imageUrl },
         trackClick,
     } = props;
 
-    if (!canRender(props.brazeMessageProps)) {
-        return null;
-    }
+    const commonStyles = selfServeStyles(props.brazeMessageProps, defaultColors);
 
     const [showBanner, setShowBanner] = useState(true);
 
-    if (!canRender(props.brazeMessageProps)) {
```

Comment:
why were there two `canRender` checks before? Mistake?

Reply:
Yep - fixed. I've also moved the check to the top of the function in each banner